src/capi.ts

```diff
@@ -48,8 +51,27 @@ const articleContributors = (content: Content): Tag[] =>
 const isImage = (elem: BlockElement): boolean =>
     elem.type === ElementType.IMAGE;
 
+const isVideo = (elem: BlockElement): boolean =>
+    elem.type === ElementType.CONTENTATOM &&
+    elem.contentAtomTypeData?.atomType === "media";
+
 const articleMainImage = (content: Content): Option<BlockElement> =>
-    fromNullable(content?.blocks?.main?.elements.filter(isImage)[0]);
+    fromNullable(content?.blocks?.main?.elements.filter(isImage)[0])
```

Comment:
You may want to apply the same logic from `articleMainVideo` here to avoid the possibility of it crashing when doing an index lookup on `undefined`.